### ISSUE TITLE
Add map of active and prospective nodes to site

### ISSUE THREAD

**Issue by dcwalk** [tomeshnet/tomesh.net#151]
Working from @darkdrgn2k's map codebase (currently accessible at http://map.tomesh.net):
https://github.com/darkdrgn2k/TOMeshNodeMap

We want to have:
- [x] Finalize Schema
- [x] Commit Initial Node List
- [x] new page for map (available at `/map`)
- [x] add Map to global navigation

**Comment by dcwalk** [comment]
Other notes about the map:
- Using google api
- node data coming from a json (node-list GitHub)
- submit from the map interface (using a form! backend: submit PR to node list)

For stage 1 we'll not spend time optimizing for monitoring needs.

Stage 2: 
- Hooking in with monitoring functionality
- "status" in a broader sense:
  - online?
  - last seen
  - node peers

**Comment by darkdrgn2k** [comment]
Submit PR request may be harder via api the first thought
https://developer.github.com/v3/pulls/#create-a-pull-request
Seems you need to first commit the change to an GitHub branch tree before you do a pull request

So the way i see it to hook it from a form the form would need to

1) Create a branch
2) Update the json in that branch
3) commit that branch
4) create a commit request against the branch

**Comment by darkdrgn2k** [comment]
Work is being done to integrate the existing code base into the tomesh site

Working tree for the map is located at https://github.com/darkdrgn2k/tomesh.net/tree/Map 

- [x] Added map as /map
- [x] Added to global nav

Just working some loose ends, formatting
Current path for nodes is https://github.com/darkdrgn2k/node-list/master/nodeList.json
- [x] Commit node list to https://github.com/tomeshnet/node-list/master/nodeList.json

**Comment by dcwalk** [comment]
Nice! 

Just to transfer over @garrying 's comment from our chat -- this work has been merged into the [`Dev` branch](https://github.com/tomeshnet/tomesh.net/tree/Dev)

**Comment by darkdrgn2k** [comment]
@dcwalk @Shrinks99 @garrying Thoughts on pushing nodeList.json into node-list so it no longer references my git hub?

PR for initial Node List at tomeshnet/node-list#6

**Comment by dcwalk** [comment]
Agree @darkdrgn2k! I left my comments there :)

**Comment by darkdrgn2k** [comment]
PR Committed  tomeshnet/node-list#6 with official node list

Now waiting on final review/add of 

- [x] Updated Look/Feel of map with @Shrinks99 
- [x] Align Map to new Schema
- [x] Decision on Add a node (Since PR are trickier all reference of "add node" currently removed and replaced with "here is the json of your node for your reference"

**Comment by dcwalk** [comment]
🎉
